pmda/test/test_util.py

```diff
@@ -130,3 +131,55 @@ def test_make_balanced_slices_ValueError(n_frames, n_blocks,
     with pytest.raises(ValueError):
         make_balanced_slices(n_frames, n_blocks,
                              start=start, stop=stop, step=step)
+
+
+@pytest.mark.parametrize('n_frames', [3, 4, 10, 19, 101, 331, 1000])
+def test_second_order_moments(n_frames):
```

Comment:
Can you think of a way to also parametrize `isplit`? Something along the lines of "make a list of 10 split points and also include 1 and -1".

Btw, does `isplit=0` work?

Reply:
isplit=0 does not work because it cannot take a slice of length zero, if I'm remembering correctly. I could generate a parameterize isplit in a similar method as in test_reduce(). I didn't add that functionality because it seemed it would be better to test that in another function, which is why I added something similar to test_reduce().